template/pnpm-workspace.yaml

```diff
@@ -0,0 +1,9 @@
+packages:
```

Comment:
Should this pnpm-specific file be included?

Reply:
Do we also need to add something similar for Yarn in that case?  It's worth keeping in mind we generally use Yarn internally.

Reply:
Turbo automatically adds the config needed for Yarn or NPM if you choose those package managers.

We still need to include this pnpm config file as it adds config to tell it which Node.JS packages should have their `postinstall` scripts run to build or download the native part of the module. It will just not be used can can be deleted if using a different package manager.